schemata/media.yaml

```diff
@@ -0,0 +1,1263 @@
+---
+id: media.json
+"$schema": "http://json-schema.org/draft-04/schema"
+title: Media
+description: "Content blocks like image, video and article are child of media, both have cover"
+type:
+- object
+definitions:
+  cover:
+    description: "A cover image that has many details about the media, useful for previews"
+    type:
+    - object
+    properties: 
+      orientation:
+        description: "Cover image orientation"
+        type: string
+        example: landscape
+      src:
+        description: "URL for the cover image"
+        type: string
+        example: "https://farm8.staticflickr.com/7614/17055279932_6e242fddd5_b.jpg"
+      altsrc:
+        description: "Original URL (if the image has a fullscale URL, the original URL will be stored here"
+        type: string
+        example: "https://farm8.staticflickr.com/7614/17055279932_6e242fddd5.jpg"
+      ratio:
+        description: "Cover image ratio"
+        type: string
+        example: "128:101"
+      aspect:
+        description: "Calculated image ration"
+        type: number
+        example: 1.2673267326732673
+      width:
+        description: "Cover image width"
+        type: integer
+        example: 1024
+      height:
+        description: "Cover image height"
+        type: integer
+        example: 808
+      faces:
+        description: "Extracted faces from the image, represented as a rectangle for each face found"
+        type: array
+        example: [[0, 100, 150, 200]]
+      colors:
+        description: "Extracted colors from the image, represented as an array of 5 RGB colors"
+        type: array
+        example: [[122,174,213],[8,12,21],[255,254,255],[75,108,141],[202,207,203]]
+      saliency:
```

Comment:
I think big/complex data like `saliency,` `color` and `faces` should be in its own definitions, and referred to. Maybe an `imagemeasurements` file? Can be done in separate pull request, but I think that would clean things up quite a bit.